pkg/scheduler/apis/config/types_pluginargs.go

```diff
@@ -181,3 +181,18 @@ type VolumeBindingArgs struct {
 	// If this value is nil, the default value will be used.
 	BindTimeoutSeconds int64
 }
+
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// NodeAffinityArgs holds arguments to configure the NodeAffinity plugin.
+type NodeAffinityArgs struct {
+	metav1.TypeMeta
+
+	// DefaultAffinity is applied to all Pods additionally to the NodeAffinity
+	// specified in the PodSpec. That is, Nodes need to satisfy DefaultAffinity
+	// AND .spec.NodeAffinity. DefaultAffinity is empty by default (all Nodes
```

Comment:
Already using `default` as the value when pod doesn't specify one somewhere, I guess most people will think about that semantic in the beginning. Maybe call it 'AdditionalNodeAffinity', then can help people to aware that is different semantic.

Reply:
That's fair. Although I find `Additional` confusing What about `ImplicitAffinity`? @Huang-Wei @ahg-g any other suggestions?

Reply:
So if a pod doesn't specify any affinity, we have a default affinity that's applied to it? If this `DefaultAffinity` is set, does that override the other default affinity? If so, then I think "Default" is fine

Reply:
No, that's not the behavior. It's `AND` semantics.

Reply:
Ahh, then yeah something besides Default is a good idea

Reply:
My preferred options are: `InherentAffinity` and `ImplicitAffinity`

Reply:
GlobalConstraints?

Reply:
I would say Global has the same problem as Default, and it might be even more confusing given that it's a per-profile configuration.

Reply:
I don't think it has the same semantics as default, but yeah, it might not work well with per-profile scope. Let me think a bit.

Reply:
The thing about implicit/implied/inherent is that it makes one pasue a little before realizing what it could mean. I am not objecting to those suggestions, but just for the sake of brain storming, anther option could be along the lines of "AddedConstraints".

Reply:
Not an expert on naming... But yes, "default" is not concise.

Another option: somehow "SupplementalAffinity" jumps out of my mind :)

Reply:
AppendedAffinity/Constraints

Reply:
I am inclined to use additional/added/supplemental adjective

Reply:
Went with `AddedAffinity`